ansible/roles/test/tasks/everflow_testbed/testcase_6.yml

```diff
@@ -0,0 +1,73 @@
+# Test case 6 - ARP/ND packets mirroring
+
+- set_fact:
+    mirror_session_name: arpnd_mirror_session
+    mirror_table_name: EVERFLOW
+    mirror_tablev6_name: EVERFLOWV6
+    mirror_acl_rule_name: RULE_ARP
+    mirror_acl_rulev6_name: RULE_ND
+
+- name: Create route with next hop {{ dst_port_1 }}.
+  shell: vtysh -e "conf t" -e "ip route {{ session_prefix_1 }} {{ neighbor_info_1['addr'] }}"
+  become: yes
+
+- block:
+  - name: Create a mirror session
+    shell: |
+      config mirror_session add {{mirror_session_name}} {{session_src_ip}} {{session_dst_ip}} {{session_dscp}} {{session_ttl}}
+    become: yes
+
+  - name: Create a rule
+    shell: |
+      redis-cli -n 4 hmset "ACL_RULE|{{mirror_table_name}}|{{mirror_acl_rule_name}}" "PRIORITY" "8888" "MIRROR_ACTION" "{{mirror_session_name}}" "ETHER_TYPE" "2054"
+    become: yes
+
+  - name: Create a v6 rule
+    shell: |
+      redis-cli -n 4 hmset "ACL_RULE|{{mirror_tablev6_name}}|{{mirror_acl_rulev6_name}}" "PRIORITY" "8887" "MIRROR_ACTION" "{{mirror_session_name}}" "ICMPV6_TYPE" "135"
+    become: yes
+
+  - name: Pause to sync the rule
+    pause: seconds=3
+
+  - name: "Start PTF runner"
+    include: roles/test/tasks/ptf_runner.yml
+    vars:
+      ptf_test_name: EVERFLOW Neighbor Test
+      ptf_test_dir: acstests
+      ptf_test_path: everflow_neighbor_test.EverflowNeighborTest
```

Comment:
This needs to match the name of the python file - I'll go ahead and change that to everflow_neighbor_test.py.